packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -389,7 +389,7 @@ export class AtomicBreadbox implements InitializableComponent {
       )
       .map(({value, facetId, field, label}) => ({
         facetId,
-        label: this.bindings.i18n.t(label ? label : 'no-label'),
+        label: label ? label : field,
```

Comment:
Why is i18n not required anymore here?

Reply:
i18n was required because of the 'no-label' value. We have a entry for no-label in locales.json. Now, since both possible values from the condition come from the index, there is no need to check if there is an entry in locales.json.